ocaml/forkexecd/helper/dune

```diff
@@ -0,0 +1,27 @@
+(rule
+ (targets vfork_helper)
+ (package forkexec)
+ (deps
+  close_from.h
+  close_from.c
+  logs.h
+  logs.c
+  syslog.h
+  syslog.c
+  redirect_algo.h
+  vfork_helper.h
+  vfork_helper.c
+  Makefile)
+ (action
+  (run make)))
+
+(rule
+ (alias install)
+ (deps vfork_helper)
+ (action (run true)))
+
+(install
+  (package vhd-tool)
```

Comment:
Shouldn't this be installed as part of the `forkexec` package?

Reply:
Yes... in theory. There is a correspondence between the final file destination and directory specified in the main `Makefile`. `vhd-tool` here is used to get the final `/usr/libexec/xapi/vfork_helper`. I'll try the suggested change.

Reply:
Directory changed from `/usr/libexec/xapi/` to `/usr/lib/xenopsd/xapi/`.

Reply:
That PATH needs changing, I don't see any reason why forkexecd's should point to a xenopsd directory

Reply:
That's why I used `vhd-tool` package. Not saying it sounds good too, just that it does its job and I don't know how to use a more correct package name but get the right `PATH`.

Reply:
Done, solved the package name issue moving `forkexec` with `vhd-tool` in `Makefile` (as discussed internally and tested).